src/node-saml/saml.ts

```diff
@@ -468,24 +465,19 @@ class SAML {
       this.signRequest(samlMessage);
     }
     Object.keys(samlMessage).forEach((k) => {
-      target.query[k] = samlMessage[k];
+      target.searchParams.append(k, samlMessage[k] as string);
```

Comment:
This doesn't seem to be the same behavior. It seems that instead of replacing the query, we are now appending to it.

Reply:
updated